contrib/epee/include/storages/portable_storage.h

```diff
@@ -91,8 +83,8 @@ namespace epee
 
       //-------------------------------------------------------------------------------
       bool		store_to_binary(byte_slice& target, std::size_t initial_buffer_size = 8192);
-      bool		load_from_binary(const epee::span<const uint8_t> target, const limits_t *limits = NULL);
-      bool		load_from_binary(const std::string& target, const limits_t *limits = NULL) { return load_from_binary(epee::strspan<uint8_t>(target), limits); }
+      bool		load_from_binary(const epee::span<const uint8_t> & target, const limits_t *limits = NULL);
```

Comment:
```suggestion
      bool		load_from_binary(const epee::span<const uint8_t>  target, const limits_t *limits = NULL);
```

I would feel better if we could revert the pass by reference, this PR should not include extra changes like this.

Reply:
Two voices are enough to convince me.
Soon the measurements will show if span.h is an outstanding bottleneck anyway.

Reply:
I don't have an opinion on reference vs copy here but this PR should not introduce changes like this so that we avoid introducing unintended changes.